src/api/model/user_model.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import BaseModel
+from datetime import datetime
+
+class Address(BaseModel):
+    street: str
+    city: str
+    state: str
+    country: str
+    postalCode: str
+
+class User(BaseModel):
+    id: int
```

Comment:
'id' is of type long, as per documentation. I am not sure what is the equivalence of long in python. Please update the field type.

![image](https://github.com/user-attachments/assets/e260931c-6020-4b66-a883-aeb53a73bbd1)